app/controllers/tests_controller.rb

```diff
@@ -0,0 +1,4 @@
+class TestsController < ApplicationController
```

Comment:
Oh wow. Something called a test under the app directory?  My mind is hurting right now. Tests and app code are different. This is a production controller named "test". Hmmm can we name them performance runs? Or something less ambiguous that means something else in the rails community?